src/main/java/io/quarkus/gizmo/BytecodeCreator.java

```diff
@@ -851,6 +850,15 @@ default void breakScope() {
      */
     WhileLoop whileLoop(Function<BytecodeCreator, BranchResult> conditionFun);
 
+    
+    /**
+     * Create a new for-each loop construct.
+     * 
+     * @param iterable
+     * @return the for-each construct
+     */
+    ForEeachLoop forEach(ResultHandle iterable);
```

Comment:
@FroMage this one is for you ;-)

And the usage is like https://github.com/quarkusio/gizmo/pull/115/files#diff-cc1aaa93f196abeaba7c90a119a54cd226d8972aa58083ed8c38a181feab4f48R24-R28

Reply:
My hero ❤️❤️❤️